live-examples/js-examples/string/string-substr.html

```diff
@@ -0,0 +1,6 @@
+<pre>
+<code id="static-js">const str = 'Mozilla';
```

Comment:
Because substr is not ES6-y, we should use `var` here: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6